pjp.tm08/src/main/java/com/subho/wipro/pjp/tm08/proj/dao/SalesDao.java

```diff
@@ -0,0 +1,96 @@
+package com.subho.wipro.pjp.tm08.proj.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Date;
+
+import com.subho.wipro.pjp.tm08.proj.bean.Sales;
+import com.subho.wipro.pjp.tm08.proj.bean.SalesReport;
+import com.subho.wipro.pjp.tm08.proj.util.DBUtil;
+
+public class SalesDao {
+
+	public int insertSales(Sales sales) {
+		Connection conn = null;
+		PreparedStatement pstmt = null;
+		String sql = "INSERT INTO TBL_SALES VALUES(?, ?, ?, ?, ?)";
+		
+		java.sql.Date sqlDate = new java.sql.Date(sales.getSalesDate().getTime());
+		
+		try {
+			conn = DBUtil.getConnection();
+			pstmt = conn.prepareStatement(sql);
+			pstmt.setString(1, sales.getSalesID());
+			pstmt.setDate(2, sqlDate);
+			pstmt.setString(3, sales.getProductID());
+			pstmt.setInt(4, sales.getQuantitySold());
+			pstmt.setDouble(5, sales.getSalesPricePerUnit());
+			
+			if (pstmt.executeUpdate() == 1) return 1;
+			else return 0;
+		} catch (SQLException e) {
+			e.printStackTrace();
+			return 0;
+		}
+	}
+	
+	public String generateSalesID(Date salesDate) {		
+		Connection conn = null;
+		PreparedStatement pstmt = null;
+		String sql = "SELECT SEQ_SALES_ID.NEXTVAL FROM DUAL";
+		
+		int SEQ_SALES_ID = 0;
+		String out = salesDate.toString().substring(salesDate.toString().length()-2, salesDate.toString().length());
+		
+		try {
+			conn = DBUtil.getConnection();
+			pstmt = conn.prepareStatement(sql);
+			ResultSet rs = pstmt.executeQuery();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always check the return of one of the navigation method (next,previous,first,last) of a ResultSet.](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5628071)